app/scripts/models/auth_brokers/base.js

```diff
@@ -263,6 +263,10 @@ define([
        */
       emailVerificationMarketingSnippet: true,
       /**
+       * Should the signup page show the `Choose what to sync` checkbox
+       */
+      showChooseWhatToSyncCheckbox: true,
```

Comment:
`show` is a modifier of the actual capability is `chooseWhatToSyncCheckbox`. Mind changing the name?

This reminds me, we should probably start documenting capabilities in an external doc. Perhaps we can generate the doc using jsdoc and a custom tag.